homeassistant/helpers/system_info.py

```diff
@@ -23,14 +23,13 @@
         "arch": platform.machine(),
         "timezone": str(hass.config.time_zone),
         "os_name": platform.system(),
+        "os_version": platform.release(),
```

Comment:
Now you're also calling this for Windows and Mac?

Reply:
Yes, but ``release()`` is just a getter and ``uname()`` is already called in ``system()``

platform sourcecode:
```py
def system():
    """ Returns the system/OS name, e.g. 'Linux', 'Windows' or 'Java'.
        An empty string is returned if the value cannot be determined.
    """
    return uname().system

def release():
    """ Returns the system's release, e.g. '2.2.0' or 'NT'
        An empty string is returned if the value cannot be determined.
    """
    return uname().release
```